sunpy/lightcurve/sources/sit.py

```diff
@@ -0,0 +1,326 @@
+# -*- coding: utf-8 -*-
+"""Provides programs to process and analyze STEREO SIT data."""
+
+# This module was developed with funding from the Google Summer of Code 2015
+
+from __future__ import absolute_import
+
+__authors__ = ["Ankit Kumar"]
+__email__ = "ankitkmr.iitk@gmail.com"
+
+from datetime import timedelta,datetime
+from calendar import monthrange
+from sunpy.lightcurve import LightCurve
+from dateutil.relativedelta import relativedelta
+from sunpy.time import parse_time, TimeRange, is_time_in_given_format
+from sunpy.util import net
+from shlex import split
+from astropy.io import ascii
+from astropy.table import Table, Column
```

Comment:
The imports should follow the [pep :8ball: import order](https://www.python.org/dev/peps/pep-0008/#imports)
>  Imports should be grouped in the following order:
>
>    standard library imports
>    related third party imports
>    local application/library specific imports
>
>You should put a blank line between each group of imports.